### PR TITLE
Fixed USDU-173 - scene not marked dirty after changing load payload flag

### PR DESCRIPTION
**Pull request by mirceaispas**
## Purpose of this PR

**Ticket/Jira #:**
https://jira.unity3d.com/browse/USDU-173: Scene is not marked dirty when loading a payload

## Testing

**Functional Testing status:**
Manual tests and automated tests for loading/unloading payloads

**Performance Testing status:**
N/A

## Overall Product Risks

**Complexity:**
Minimal

**Halo Effect:**
Minimal

## Additional information

**Note to reviewers:**

**Reminder:**

- [ ] Add entry in CHANGELOG.md _(if applicable)_

### CODE REVIEW

**Review by IlliaKo — APPROVED** [review]
Auto-test looks good. 
Played around fixed case, loading/unloading payloads, looks good :)

**Review by vickycl — APPROVED** [review]
LGTM, nice test coverage :)

**Comment by mirceaispas** [comment]
Rebased on dev to include CI fixes

File: package/com.unity.formats.usd/Tests/Editor/USDPayloadComponentTests.cs [hunk 1]
```diff
@@ -192,5 +193,39 @@ public IEnumerator IsLoaded_IsFalse_WhenIsUnloadedFromComponent()
             Assume.That(usdPayload, Is.Not.Null, "Could not find USDPayload in the subtree.");
             Assume.That(usdPayload.IsLoaded, Is.False, "UsdPayload.IsLoaded should be set to false.");
         }
+
+        [UnityTest]
+        public IEnumerator ChangingUsdPayloadSateFromUnloadedToLoaded_MarksSceneDirty()
```
Comment (michaeljblain) [inline package/com.unity.formats.usd/Tests/Editor/USDPayloadComponentTests.cs]: Minor typo: Sate -> State

File: package/com.unity.formats.usd/Tests/Editor/CleanTestEnvironment.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
+using NUnit.Framework;
```
Comment (michaeljblain) [inline package/com.unity.formats.usd/Tests/Editor/CleanTestEnvironment.cs:15]: We should add a copyright notice/license at the top of the file.

**Review by michaeljblain — CHANGES_REQUESTED** [review]
Looks good to me. Some minor suggestions.

**Review by michaeljblain — APPROVED** [review]

**Pull request merged**